src/renderers/dom/shared/__tests__/ReactDOMComponent-test.js

```diff
@@ -148,6 +148,19 @@ describe('ReactDOMComponent', function() {
       expect(console.error.argsForCall.length).toBe(2);
     });
 
+    it('should warn for unknown prop', function() {
+      spyOn(console, 'error');
+      var container = document.createElement('div');
+      ReactDOM.render(<div foo="bar" />, container);
+      expect(console.error.argsForCall.length).toBe(1);
+      expect(console.error.argsForCall[0][0]).toContain(
+        'Warning: Unknown prop foo.  See https://fb.me/react-unknown-prop'
```

Comment:
Let’s make the warning more descriptive, similar in style to the other warnings being tested in this file?